cli/test-suites-and-reports.md

```diff
@@ -12,6 +14,18 @@ maestro test myFolderWithTests/
 
 Maestro will run every flow from the directory _excluding subfolders_. The command will complete successfully if and only if all the Flows have been completed successfully.
 
+#### Using a list of flows
+
+Maestro can run a list of tests and generate a test report at the end.
+
+To run a list of tests, pass to `maestro test` the Flows paths to the Flows you want to run separated by spaces:
+
+```
+maestro test "myFolderWithTests/test1.yaml myFolderWithTests/test2.yaml"
```

Comment:
Are the quotes required?

Reply:
Good call, no, no needed. Let me update it.